### PR TITLE
Sample details refactor/ documentation

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
* [X] User documentation updated for UI or technical changes.

### CODE REVIEW

**Comment by joshsadam** [comment]
So I have a small problem with your screen shots.  It is hard to tell they are modals by the way you have them cropped.  @ericenns and I have been working on a ton of screen shots recently, and for all of them, including modals, we like to capture the whole screen at 1600px.  This prevent modal or smaller screen areas from totally blowing up when viewed.

File: doc/user/user/samples/index.md [hunk 1]
```diff
@@ -31,21 +31,25 @@ The samples listing shows high-level sample details, such as:
 
 ### Viewing individual sample details
 
+**Note: Details for individual samples can be viewed from anywhere a sample is listed (such as from the recent activity, analysis results, analysis output files, sharing of samples, etc) by clicking on the sample name. This will launch the sample details viewer. Only a user with the project Manager role will be able to modify sample details, metadata, and files.**
+
+![Sample details page.]({{ site.baseurl }}/images/tutorials/common/samples/sample-detail-viewer.png)
```
Comment (joshsadam) [inline doc/user/user/samples/index.md:36]: I feel this image is out of place here.  You are showing the user the contents of the viewer without initially showing them how to open it, which is the subsequent image

**Comment by deepsidhu85** [comment]
> So I have a small problem with your screen shots. It is hard to tell they are modals by the way you have them cropped. @ericenns and I have been working on a ton of screen shots recently, and for all of them, including modals, we like to capture the whole screen at 1600px. This prevent modal or smaller screen areas from totally blowing up when viewed.

As discussed the docs are getting a complete overhaul so we are going to leave as is for now

File: doc/user/user/samples/index.md (hunk 1 above)
Comment (joshsadam) [reply doc/user/user/samples/index.md:36]: After discussing with @deepsidhu85 this will be updating in a future documentation upgrade.

**Pull request merged**